fe1-web/src/core/components/PoPIconButton.tsx

```diff
@@ -15,18 +15,26 @@ const SIZE_MAP = {
 const PoPIconButton = (props: IPropTypes) => {
   const { onPress, buttonStyle, disabled, toolbar, size, testID, name } = props;
 
+  let iconColor = Color.contrast;
+
+  if (buttonStyle === 'secondary') {
+    // in case of an outlined button, the icon color
+    // should be the same as the border's
+    if (disabled) {
+      iconColor = Color.inactive;
```

Comment:
is the `inactive` color only useful for the `secondary` button style, or should it be used for all button tyles ?

Reply:
The primary button (colored background, white icon) also has a disabled state but there only the background color changes which is done in `PoPButton`. But for the secondary button style (colored icon, outlined button) it is necessary to change the color of the icon to gray. Does this explanation make sense?